docs/REST API/rest-api.md

```diff
@@ -7137,6 +7269,36 @@ Returns the information of the specified software title. By default `versions` a
     "title": "Firefox.app",
     "source": "apps",
     "hosts_count": 48,
+    "managed": [
```

Comment:
I think we can make this an object instead of an array and only return `managed` if a `team_id` query param is specified. 

If you agree, I think we need to add a `team_id` query param.

Why? I think it's simpler. I don't think we'll ever have more than one software item per team. 

See Loom here for more feedback:

Reply:
Updated